src/exceptions.cc

```diff
@@ -67,14 +68,15 @@ Local<Value> ErrnoException(Isolate* isolate,
 static Local<String> StringFromPath(Isolate* isolate, const char* path) {
 #ifdef _WIN32
   if (strncmp(path, "\\\\?\\UNC\\", 8) == 0) {
+    v8::NewStringType type = v8::NewStringType::kNormal;
```

Comment:
Doesn’t look like this is in the right place. There are two more instances of `NewFromUtf8` below.

Reply:
Ooh, this one was unfinished. Thanks, will change this.

Reply:
I think this is unused now? Either way, I think it makes sense to keep these inline in the code (i.e. not create separate variables)

Reply:
Okay, will make appropriate changes.